db/migrate/20160418211433_add_snapcode_image_to_developers.rb

```diff
@@ -0,0 +1,5 @@
+class AddSnapcodeImageToDevelopers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :developers, :snapcode_image, :string
```

Comment:
ah! sorry about that. those migration were for when i was locally trying to save images. i'll rebase now.